### DISCUSSION TITLE
How to get the trace when kafka messages is re-consumed

### DISCUSSION THREAD

**Discussion by Linda-pan** [apache/skywalking#9474] [Q&A]
How to get the trace when kafka messages is re-consumed？

We  made a schedule to re-consume failed the kafka message from rockDB，and we found it was missing traceId. I would like to inquire about how to get traceId. We can attach some custom information to the kafka message header.

**Comment by wu-sheng** [comment]
I am not sure about your case. Why this should be SkyWalking community's concern?
